Slim/Menu/BrowseLibrary/Releases.pm

```diff
@@ -162,10 +166,22 @@ sub _releases {
 		!$primaryReleaseTypes{$_};
 	} keys %releaseTypes);
 
+	my $limitList = sub {
+		my $list = shift;
+		my $name = shift;
+		my $albumCount = scalar @$list;
+		if ( $albumCount > LIST_LIMIT ) {
+			splice @$list, LIST_LIMIT;
+			$name .= ' (' . cstring($client, 'FIRST') . ' ' . LIST_LIMIT . ' ' . cstring($client, 'ALBUMS') .')';
+		}
+		return $list, $name;
```

Comment:
As `$list` is a list _reference_, you actually don't have to return it. It's modified directly. Therefore `($listRef, $name) = $limitList($listRef, $name);` is overwriting `$listRef` with itself, like `$a = $a`. You don't have to return `$list`.

Unless I'm completely wrong, of course.